.github/workflows/tests_gpu_cpp.yml

```diff
@@ -17,11 +17,10 @@ on:
       - synchronize
       - ready_for_review
     paths:
-      - .github/workflows/tests_lgpu_cpp.yml
+      - .github/workflows/tests_gpu_cpp.yml
       - pennylane_lightning/core/src/**
       - '!pennylane_lightning/core/src/simulators/lightning_kokkos/**'
       - '!pennylane_lightning/core/src/simulators/lightning_qubit/**'
```

Comment:
We also need to remove these paths to be able to use the label even when the PR doesn't change any of these paths
```suggestion
```

Reply:
I was not aware of this intention. I will make sure to update accordingly.